app/src/main/res/values/strings.xml

```diff
@@ -372,15 +372,31 @@ Please report any problems you encounter via Github.</string>
     <string name="toast_invalid_environment_variables">Value is not a valid environment variable string</string>
 
     <!-- Title for the preference to reset Syncthing indexes -->
-    <string name="streset_title">Reset Database</string>
+    <string name="stresetdatabase_title">Reset Database</string>
 
     <!-- Syncthing was reset -->
-    <string name="streset_question">This action should only be performed based on a recommendation from our support team.
-        \nAre you sure you want to clear Syncthing\'s index database?
+    <string name="stresetdatabase_question">This action should only be performed based on a recommendation from our support team.
+        \nAre you sure you want to clear Syncthing\'s database?
+    </string>
+
+    <!-- to be removed after merge into the master -->
+    <string name="streset_title">to be removed</string>
+    <string name="streset_question">to be removed</string>
+    <string name="streset_done">to be removed</string>
```

Comment:
Why not remove now?

Reply:
Hi,
thanks for taking time.
Removing the strings in the en-lang causes the build to fail on travis and teamcity.
I would have to remove them in all lang files manually one by one for the build to be okay.
Which way is correct - leaving the build failing or going through all 36 lang files?
I tried to create a transifex branch (that can merge later synced with the github merge) but it says my access level is not sufficient (Error forbidden) - is this a better way to do it?
I wonder how @Nutomic's procedure was when strings had to be removed.

Reply:
There might be a command to regenerate transifex stuff. What's the failure when you remove strings?

Reply:
I believe the right thing to do is not rename the existing strings. The proposed keys might be better, but it means all translations must be done again. I.e. Just add the `stresetdelta...` strings and leave the existing `streset...` alone.

Reply:
I'll post the error tomorrow and then make a commit with resetdb renamed back to streset.

Reply:
Android Studio has a "translation editor" which allows you to remove all translations for a single string. Just open any strings.xml file in Android Studio, and it should show a notification at the top of the editor. Then select the string you want to remove, and press the red minus at the top.

Reply:
Thanks for the hint @Nutomic. I chose to remove the original streset strings because they referred to "index" in lang-en. This can be confusing, thus just a little bit, if talking about index and delta index on two UI buttons. I think its more straightline if we stick to syncthing doc's and call it database / delta indexes reset. Commit https://github.com/syncthing/syncthing-android/pull/1061/commits/8a33d007fc371ce4f849d4dff7445ba0682b8943